### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by falguniislam**
## Javascript--PROJECT: RESTAURANT PAGE

Let’s use what we’ve learned and take a chance to continue practicing DOM manipulation - dynamically rendering a simple restaurant homepage! By the end, we are going to be using only JavaScript to generate the entire contents of the website!

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+markTab('about')
+
+
+// Tabs handler
+const tabNavigation = document.querySelectorAll('.tab-title')
+tabNavigation.forEach((tab) => {
+    tab.addEventListener('mousedown', tabToggle)
+})
+
+function tabToggle(tab) {
```
Comment (JamesOkunlade) [inline src/index.js]: Change this to `es6 arrow function`.

Comment (falguniislam) [reply src/index.js]: tabToggle (tab) => {
 const tabs = {
        'about': About.getTab(),
        'menu': Menu.getTab(),
        'find-us': FindUs.getTab()
 }
like this.

Comment (falguniislam) [reply src/index.js]: tabToggle (tab) => {
const tabs = {
'about': About.getTab(),
'menu': Menu.getTab(),
'find-us': FindUs.getTab()
}
like this.

Comment (AdnanAfsari) [reply src/index.js]: Yes like this 😄

**Comment by AdnanAfsari** [comment]
Hello @falguniislam ✋ 

There is still a  linter error and it says ./eslint.config does not exists.

I think you have deleted it and created a new config which is .eslintrc.json so you should include it in your .stickler.yml.

Please do that and fix errors if there is any and submit for another review 😄.

Happy Coding

**Comment by falguniislam** [comment]
Hello i am having problem to understand eslint config file,  can you help
me please. I tried a lot but still i did not understand it.

**Pull request merged**